src/components/registration-view/registration-view.jsx

```diff
@@ -9,19 +9,65 @@ import {
   Col,
   Row,
 } from "react-bootstrap";
+import { Link, useHistory } from "react-router-dom";
 
-//import "./registration-view.scss";
 import axios from "axios";
+import { indexOf } from "lodash";

```

Comment:
I will suggest to remove this, it is probably comes from the autocomplete feature in vs code. thanks!

Reply:
Thanks, just fixed it. :)